packages/api/src/hooks/p2p/index.ts

```diff
@@ -0,0 +1 @@
+export { default as useP2PAdvertiserInfo } from './useP2PAdvertiserInfo';
```

Comment:
this has needs to be handled for entities not for each and every hook, 
the reason is that moving forward we need to import/export the entities and not the calls.
with this approach we will have circular deps.

Reply:
So for now I remove these imports later we can handle them 👍